README.md

```diff
@@ -156,6 +156,13 @@ Below you can find information on how to push local code changes to a kind clust
 - update manifests to use the generated image: `IMAGE_REGISTRY=localhost:5000/maiqueb make manifests`
 - deploy the controller: `kubectl apply -f manifests/dynamic-networks-controller.yaml`
 
+### Mapping a container image to the code
+To know which git commit ID is in a certain container image perform the following steps:
+```bash
+podman inspect ghcr.io/k8snetworkplumbingwg/multus-dynamic-networks-controller:latest-amd64 -f '{{index .Labels "multi.GIT_SHA"}}'
```

Comment:
note that if you use `.Config.Labels`
it will work for both podman and docker (even that if you look on the inspect output, the Config doesn't have it visible)